bridge.go

```diff
@@ -101,6 +103,17 @@ func (b *Bridge) pairAs(appName string) error {
 	if err != nil {
 		return err
 	}
+
```

Comment:
You're absolutely right, I've checked the [documentation](http://www.developers.meethue.com/documentation/configuration-api#71_create_user) and the total limit is indeed 40 characters. Additionally, it is also specified that the `application_name` is maximum 20 chars and the `devicename` is 19.

How about something like this instead?

```go
if len(appName) > 20 {
    appName = appName[:20]
}

deviceName := fmt.Sprintf("%s-%s", host, runtime.GOOS)
if len(deviceName) > 19 {
    deviceName = deviceName[:19]
}

msg, err := b.call(http.MethodPost, map[string]interface{}{
	"devicetype": fmt.Sprintf("%s#%s", appName, deviceName),
})
```
This should make sure that we're 100% on track with the specs.

It would of course be possible to additionally warn the user that the entry has been truncated, but if we do decide to do that, I was thinking to implement a _verbose_ flag (`-v` or `--verbose`) which would trigger this behavior. CLIs with too much